adapter/stdioLogger/stdioLogger.go

```diff
@@ -22,33 +22,32 @@ import (
 	"os"
 
 	"istio.io/mixer/adapter/stdioLogger/config"
-	"istio.io/mixer/pkg/aspect"
-	"istio.io/mixer/pkg/aspect/logger"
+	"istio.io/mixer/pkg/adapter"
+	"istio.io/mixer/pkg/adapter/logger"
 	"istio.io/mixer/pkg/registry"
 
 	me "github.com/hashicorp/go-multierror"
 )
 
 type (
-	adapter    struct{}
-	aspectImpl struct {
+	adapterState struct{}
+	aspectImpl   struct {
 		logStream io.Writer
 	}
 )
 
 // Register adds the stdioLogger adapter to the list of logger.Aspects known to
 // mixer.
-func Register(r registry.Registrar) error { return r.RegisterLogger(&adapter{}) }
+func Register(r registry.Registrar) error { return r.RegisterLogger(&adapterState{}) }
 
-func (a *adapter) Name() string { return "istio/stdioLogger" }
-func (a *adapter) Description() string {
+func (a *adapterState) Name() string { return "istio/stdioLogger" }
```

Comment:
isn't `state` implied even if you use just `adapter`. It is the receiver of these methods so it will hold configuration and other state.